ee/vellum_ee/workflows/display/workflows/get_vellum_workflow_display_class.py

```diff
@@ -24,9 +24,4 @@ def get_workflow_display(
         except IndexError:
             return base_display_class(workflow_class)
 
-    if not issubclass(workflow_display_class, base_display_class):
```

Comment:
@dvargas92495 I had to remove this because there are instances where we want to retrieve a subworkflow's display class and that display class won't be a subclass of the outer workflow's display class.

I might be using this utility wrong though. Thoughts?

Reply:
It's possible that with my suggestion above of including `from .workflow import *` this will go back to working without the need for this change. Not sure though.

Reply:
My understanding of the usage of this method was that `base_display_class` represents a common _base_ used throughout serialization, not strictly the exact display class the workflow uses.

However, if Noa's suggestion doesn't work out and removing this passes all tests with no obvious counterexample test to make, then fire away 👍